src/extension/background-script/router.ts

```diff
@@ -85,7 +85,7 @@ const routes = {
   public: {
     webbtc: {
       onboard: onboard.prompt,
-      enable: allowances.enable,
+      enable: allowances.mnemonicEnable,
```

Comment:
move those under the webbtc/nostr/etc. namespaces then we also indicated there that those are called from the window providers

Reply:
moved enable prompts to their respective namespaces